README.md

```diff
@@ -20,6 +20,10 @@ without forcing Consul users to do a full Consul upgrade.
     This enables Kubernetes to easily access external services and for
     non-Kubernetes nodes to easily discover and access Kubernetes services.
     _(Requires Consul 1.1+)_
+    
+  * [**Consul Service Mesh (Connect)**](https://www.consul.io/docs/platform/k8s/connect.html):
+    Run Consul Service Mesh (aka Consul Connect) on Kubernetes. This feature
+    injects Envoy sidecars and registers your Pods with Consul.
```

Comment:
Should we add a minumum required Consul version here as well?

Reply:
Good idea. Will do.